typescript/simple-mercari-web/src/App.css

```diff
@@ -1,44 +1,136 @@
+:root {
+  --alias-color-text-primary-default: #333;
+  --alias-color-background-secondary-default: #f5f5f5;
+  --alias-color-border-primary-default: #ccc;
+  --alias-color-border-primary-highlight: #e5e5e5;
+}
+
 .App {
   text-align: center;
 }
 
-.Title {
-  background-color: #222427;
+.Listing {
   min-height: 8vh;
   display: flex;
   flex-direction: column;
   align-items: center;
   justify-content: left;
   font-size: calc(10px + 1vmin);
-  color: white;
+  color: #222427;
+  border-bottom: 1px solid var(--alias-color-border-primary-highlight);
 }
-
-.Listing {
-  background-color: #222427;
+.ItemList {
```

Comment:
I'm sorry if I overlooked, in `typescript/simple-mercari-web/src/components/ItemList/ItemList.tsx` you changed class name of ItemList to item, so `.ItemList` is maybe no longer used.